src/services/BavService.ts

```diff
@@ -297,24 +297,24 @@ export class BavService {
 		}
 	}
 
-	async saveHmrcUuid(sessionId: string, hmrcUuid: string): Promise<void> {
-		this.logger.info({ message: `Updating ${this.tableName} table with hmrcUuid`, hmrcUuid });
+	async saveVendorUuid(sessionId: string, vendorUuid: string): Promise<void> {
```

Comment:
uuid is the type, what does the uuid represent?

Reply:
Literally a unique ID which we can use to track the specific user request to the 3rd party

Reply:
Like a correlation id? Or you you mean it is unique to each vendor? I'm guessing the latter from reading the rest of the PR. If the type is UUID but the purpose is a UID/ID I'd prefer we name the variable that way - unless the rest of the internet recognises that vendor by that UUID :P

Reply:
Unique to each user session with that vendor - so I guess it's a UID of type UUID?